docs/reference-architectures/ai/mlops-python.md

```diff
@@ -57,9 +57,13 @@ The next sections describe each of these pipelines.
 
 The CI pipeline gets triggered every time code is checked in. It publishes an updated Azure Machine Learning pipeline after building the code and running a suite of tests. The build pipeline consists of the following tasks:
 
-- **Unit test.** These tests make sure the code works and is stable.
+- **Code quality.** These tests ensure that the code conforms to the standards of the team.
 
-- **Data test.** These tests verify that the data samples conform to the expected schema and distribution. Customize this test for other use cases and run it as a separate data sanity pipeline that gets triggered as new data arrives. For example, move the data test task to a data ingestion pipeline so you can test it earlier.
+- **Unit test.** These tests make sure the code works, has adequate code coverage, and is stable.
+
+- **Data test.** These tests verify that the data samples conform to the expected schema and distribution. Customize this test for other use cases and run it as a separate data sanity pipeline that gets triggered as new data arrives. For example, move the data test task to a *data ingestion pipeline* so you can test it earlier.
+
+**Note:** CI/CD concepts for data ingestion pipelines are not covered in this article. For more information on the architecture and best practices for CI/CD of a data ingestion pipeline, see [DevOps for a data ingestion pipeline](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-cicd-data-ingestion).
```

Comment:
Consider changing the first sentence to: "You should consider enabling DevOps practices for the data used to train the machine learning models, but this is not covered in this article."